ecchronos-binary/src/pylib/ecchronoslib/table_printer_v2.py

```diff
@@ -0,0 +1,126 @@
+
+        for vnode_state in sorted_vnode_states:
+            _add_vnode_state_to_table(vnode_state, vnode_state_table)
+
+        table_formatter.format_table(vnode_state_table)
+
+def _add_vnode_state_to_table(vnode_state, table):
+    entry = [vnode_state.start_token, vnode_state.end_token, ', '.join(vnode_state.replicas),
+             vnode_state.get_last_repaired_at(), vnode_state.repaired]
+
+    table.append(entry)
+
+def print_summary(repair_jobs):
+    status_list = [job.status for job in repair_jobs]
+    summary_format = "Summary: {0} completed, {1} on time, {2} blocked, {3} late, {4} overdue"
+    print(summary_format.format(status_list.count('COMPLETED'),
+                                status_list.count('ON_TIME'),
+                                status_list.count('BLOCKED'),
+                                status_list.count('LATE'),
+                                status_list.count('OVERDUE')))
+
+def print_repair_summary(repair_jobs):
+    status_list = [job.status for job in repair_jobs]
+    summary_format = "Summary: {0} completed, {1} in queue, {2} blocked, {3} warning, {4} error"
+    print(summary_format.format(status_list.count('COMPLETED'),
+                                status_list.count('IN_QUEUE'),
+                                status_list.count('BLOCKED'),
+                                status_list.count('WARNING'),
+                                status_list.count('ERROR')))
+
+
+def print_schedules(repair_jobs, max_lines):
+
+    repair_jobs_table = [["Id", "Keyspace", "Table", "Status", "Repaired(%)",
+                          "Completed at", "Next repair"]]
+    print("Snapshot as of", datetime.now().strftime("%Y-%m-%d %H:%M:%S"))
+    print_schedule_table(repair_jobs_table, repair_jobs, max_lines)
+    print_summary(repair_jobs)
+
+def print_repairs(repair_jobs, max_lines):
+    repair_jobs_table = [["Id", "Keyspace", "Table", "Status", "Repaired(%)",
+                          "Completed at"]]
+    print_repair_table(repair_jobs_table, repair_jobs, max_lines)
+    print_repair_summary(repair_jobs)
+
+def print_schedule_table(repair_jobs_table, repair_jobs, max_lines):
+
+    sorted_repair_jobs = sorted(repair_jobs, key=lambda job: job.last_repaired_at_in_ms, reverse=True)
+    if max_lines > -1:
+        sorted_repair_jobs = sorted_repair_jobs[:max_lines]
+
+    for repair_job in sorted_repair_jobs:
+        repair_jobs_table.append(_convert_schedule_job(repair_job))
+    table_formatter.format_table(repair_jobs_table)
+
+def print_repair_table(repair_jobs_table, repair_jobs, max_lines):
+
+    sorted_repair_jobs = sorted(repair_jobs, key=lambda job: job.completed_at, reverse=True)
+    if max_lines > -1:
+        sorted_repair_jobs = sorted_repair_jobs[:max_lines]
+
+    for repair_job in sorted_repair_jobs:
+        repair_jobs_table.append(_convert_repair_job(repair_job))
+    table_formatter.format_table(repair_jobs_table)
+
+def print_repair_job(repair_job):
```

Comment:
Not sure where this is used but please remove "job", use either repair or schedule